ds-prep-python-lists/main.py

```diff
@@ -0,0 +1,56 @@
+# 3
+ages = [29, 19, 17, 26, 32, 16, 21]
+
+# 4
+print(ages)
+
+# 6
+print('Fifth element in list Ages: {}'.format(ages[4]))
+
+# 7
+print('Second to last element in list Ages: {}'.format(ages[1:]))
```

Comment:
This should _only_ get one element (the second-to-last element), and it should use a negative index:

---

![image](https://user-images.githubusercontent.com/16143771/148817895-e8f8dc44-729a-49b3-92d6-9b019698be5a.png)

Reply:
Hi Tim, I am not sure how to update the change here